scripts/src/saforcertadmin/push_secrets.py

```diff
@@ -0,0 +1,75 @@
+from base64 import b64encode
+from nacl import encoding, public
+import logging
+import os
+import sys
+import requests
+import argparse
+
```

Comment:
As this is not invoked from the workflow I would like to see usage instructions added to the script.

Reply:
Added usage instructions in top of the script.